.github/workflows/edfi_paging_test.yml

```diff
@@ -27,34 +27,22 @@ jobs:
         LOCK_FILE: src/edfi-paging-test/poetry.lock
```

Comment:
[Example of a passing test in my fork](https://github.com/stephenfuqua/Suite-3-Performance-Testing/actions/runs/2386742891)